src/aliceVision/keyframe/KeyframeSelector.cpp

```diff
@@ -21,6 +22,19 @@ namespace fs = boost::filesystem;
 namespace aliceVision {
 namespace keyframe {
 
+
+/**
+ * @brief Get a random int in order to generate uid
```

Comment:
```
@brief Get a random int in order to generate uid.
@warning The random don't use a repeatable seed to avoid conflicts between different launches on different data sets.
```